code/createPowerTable.m

```diff
@@ -0,0 +1,57 @@
+function [powerTable]  = createPowerTable(folderPath)
```

Comment:
Instead of declaring a new variable that takes all the data out of Header or PowerDomainData, consider using the dot notation to grab the data. This suggestion will make the code look messy, but it would save memory space. This does not need to be done before this is pulled, but it would be a nice performance update in the future.

Reply:
I should note that declaring a new variable that pulls data out from a moderately sized struct is fine. It is just an issue when you have 50k or more elements.

Reply:
I suggest we skip this for now - save for improvements later